package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nearform/heap-profiler",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
Generally I've seen we don't update the version along with the change but not sure if that applies for this repo as well. Should we check with someone?

Reply:
I think we never established a convention. But probably you're right, we should not update this here. Reverting.